### ISSUE TITLE
SQL comment inludes next line if blank, causing syntax error

### ISSUE THREAD

**Issue by 2019-05-10** [dbeaver/dbeaver#10310]
#### System information: 
- Operating system (distribution) and version Linux Debian/SID
- DBeaver version Version 7.2.4.202011011603
- Additional extensions

#### Connection specification:
- Database name and version
- Driver name
- Do you use tunnels or proxies (SSH, SOCKS, etc)? 

#### Describe the problem you're observing:
Multiple queries in one editor separated only by empty lines.
```
SELECT * FROM foo

SELECT * FROM bar
```
Position cursor at the first query and hit [CTRL]+[ENTER]
Query is successfully executed

Add comment below the first query
```
SELECT * FROM foo
--

SELECT * FROM bar
```
Position cursor at the first query and hit [CTRL]+[ENTER]
Query fails:
```
SQL Error [42601]: ERROR: syntax error at or near "SELECT"
  Position: 24
```
Add yet another blank line after comment
```
SELECT * FROM foo
--


SELECT * FROM bar
```
Position cursor at the first query and hit [CTRL]+[ENTER]
Query is successfully executed.

Apparently the comment also masks the next blank line

**Comment by uslss** [comment]
thanks for the report

**Comment by uslss** [comment]
seems the issue is fixed in the scope of #10416 
close this issue as duplicate
